pkgs/misc/vscode-extensions/rust-analyzer/build-deps/package.json

```diff
@@ -6,20 +6,20 @@
     "node-fetch": "^2.6.1",
     "vscode-languageclient": "^7.1.0-next.5",
     "@rollup/plugin-commonjs": "^17.0.0",
-    "@rollup/plugin-node-resolve": "^11.0.0",
-    "@types/glob": "^7.1.3",
-    "@types/mocha": "^8.0.4",
-    "@types/node": "~12.12.6",
-    "@types/node-fetch": "^2.5.7",
-    "@types/vscode": "^1.53.0",
-    "@typescript-eslint/eslint-plugin": "^4.9.0",
-    "@typescript-eslint/parser": "^4.9.0",
-    "eslint": "^7.15.0",
+    "@rollup/plugin-node-resolve": "^13.0.0",
+    "@types/glob": "^7.1.4",
+    "@types/mocha": "^8.2.3",
+    "@types/node": "~14.17.5",
+    "@types/node-fetch": "^2.5.11",
+    "@types/vscode": "^1.57.0",
+    "@typescript-eslint/eslint-plugin": "^4.28.2",
+    "@typescript-eslint/parser": "^4.28.2",
+    "eslint": "^7.30.0",
     "glob": "^7.1.6",
-    "mocha": "^8.2.1",
-    "rollup": "^2.34.2",
-    "tslib": "^2.0.3",
-    "typescript": "^4.1.2",
+    "mocha": "^9.0.2",
+    "rollup": "^2.51.1",
+    "tslib": "^2.3.0",
+    "typescript": "^4.3.5",
     "typescript-formatter": "^7.2.2",
     "vsce": "=1.88.0",
```

Comment:
I do not know if it matters, but this is [different from the version specified upstream](https://github.com/rust-analyzer/rust-analyzer/blob/2021-07-26/editors/code/package.json#L59). How is this file generated / what is it meant to track?

Reply:
It's generated by the update script.